### PR TITLE
updated README

### PR DESCRIPTION
**Pull request by ritik-malik**
This is an initial update of README, with a rough layout.
The content is expected to be modified/updated as we proceed with the project

Signed-off-by: ritik-malik <ritik18406@iiitd.ac.in>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,74 @@
...
+A rough walkthrough of the automation process:
+ 
+* Load the yml conf file
+* Load the side scripts
+* Cleanup test_env directory
+* Clone the WG repositories
+* Generate relative paths to metric markdown files in various WGs
+* Pull all the metrics in base folder
+* Make a common images directory
+* Convert metrics markdown to tex
```
Comment (yash2002109) [inline README.md:35]:
```suggestion
* Decrease heading levels (only in metric markdowns) by appending `##`
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,74 @@
...
+* Decrease heading levels (only in metric markdowns) by appending `##`
```
Comment (yash2002109) [inline README.md:36]:
```suggestion
* Convert metrics markdown to tex
```

**Review by yash2002109 — COMMENTED** [review]
Just a small change in the order of walkthrough, we would first decrease the heading levels in markdown and then convert to latex

File: README.md (hunk 2 above)
Comment (ritik-malik) [reply README.md:35]: Done

**Pull request merged**